api/server/sdk/diags.go

```diff
@@ -0,0 +1,28 @@
+package sdk
```

Comment:
New files need apache lic. That's the last comment

Reply:
added